### PR TITLE
mir-graphics-drivers-android-caf: change platform dependency to caf

### PR DESCRIPTION
**Pull request by Vince1171**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -136,7 +136,7 @@ Architecture: i386 amd64 armhf arm64
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${misc:Depends},
-         mir-platform-graphics-android15,
+         mir-platform-graphics-android-caf15,
          mir-client-platform-android5,
```
Comment (mariogrip) [inline debian/control]: this also needs to depend on caf :)

**Comment by Vince1171** [comment]
@mariogrip voilà voilà :)

**Review by mariogrip — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (Vince1171) [reply debian/control]: On cat? ^^

Comment (mariogrip) [reply debian/control]: > On cat? ^^

i mean caf :P

Comment (Vince1171) [reply debian/control]: And how to do that? ^^
Should I add android-caf to the dependency?

Comment (mariogrip) [reply debian/control]: Replace `mir-client-platform-android5`  with `mir-client-platform-android-caf5`

**Comment by mariogrip** [comment]
Thanks :)

**Pull request merged**